### PR TITLE
Improve startup time with lazy loading some args

### LINKED ISSUE #1211: Lazily load pygments styles
We currently try to retrieve all pygments styles at once, but this operation is very slow due to the import of `pkg_resources` on the pygments side.
https://github.com/httpie/httpie/blob/cfcd7413d1afbabc5ab3c88a892851335be4944a/httpie/output/formatters/colors.py#L31 

We should load these styles lazily when a user specifies `--style` explicitly.

### PR DESCRIPTION
**Pull request by isidentical**
Fixes #1211

### CODE REVIEW

**Review by jakubroztocil — CHANGES_REQUESTED** [review]
Can we have a changelog entry with some numbers + tests for the lazy loading please?

**Comment by isidentical** [comment]
> Can we have a changelog entry with some numbers + tests for the lazy loading please?

I am not really sure if this can be done through any unit tests, but I'll try add some unit tests to verify the behavior.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added support for receving multiple HTTP headers with the same name, individually. ([#1207](https://github.com/httpie/httpie/issues/1207))
 - Added support for keeping `://` in the URL argument to allow quick conversions of pasted URLs into HTTPie calls just by adding a space after the protocol name (`$ https ://pie.dev` → `https://pie.dev`). ([#1195](https://github.com/httpie/httpie/issues/1195))
 - Added support for basic JSON types on `--form`/`--multipart` when using JSON only operators (`:=`/`:=@`). ([#1212](https://github.com/httpie/httpie/issues/1212))
+- Improved startup time by 40% with lazily loading pygments plugins. ([#1211](https://github.com/httpie/httpie/pull/1211))
 - Added support for `bearer` authentication method ([#1215](https://github.com/httpie/httpie/issues/1215)).
 
+
```
Comment (jakubroztocil) [inline CHANGELOG.md:15]:
```suggestion
```

**Comment by jakubroztocil** [comment]
🚤

**Pull request merged**